src/main/java/com/springprojects/mailmanager/MailManagerApplication.java

```diff
@@ -5,7 +5,7 @@
 
 @SpringBootApplication
 public class MailManagerApplication {
-//    branche żeby sprawdzić
+//    branche żeby sprawdzić jeszcze raz
```

Comment:
No idealnie, mordeczko!
```suggestion
//    branche żeby sprawdzić jeszcze raz, jeszcze raz
```